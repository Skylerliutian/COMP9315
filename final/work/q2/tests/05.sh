./bnl 6 3 5
