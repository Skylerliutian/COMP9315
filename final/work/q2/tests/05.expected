Running: ./bnl 6 3 5

Frames:     [00] [01] [02] [03] [04]
Contents:    R04  S00  S01  S02  R05
PinCount:      0    0    0    0    0
Popularity:    2    3    3    3    2
Clock: 0

#requests: 24
#releases: 24
#hits    : 15
#reads   : 9
